### PR TITLE
[Torch] rename project to TorchBlade

### PR DESCRIPTION
**Pull request by fortianyou**
The project name TorchAddons is less specific. This commit rename the project name to TorchBlade.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,5 +1,5 @@
-[submodule "pytorch_addons/third_party/googletest"]
-	path = pytorch_addons/third_party/googletest
+[submodule "pytorch_blade/third_party/googletest"]
```
Comment (yangjunpro) [inline .gitmodules:1]: So where is the repo of pytorch_blade?

Comment (fortianyou) [reply .gitmodules:1]: `pytorch_blade` is now a folder of BladeDISC

**Review by linearhit — APPROVED** [review]
LGTM

**Review by Yancey1989 — COMMENTED** [review]
#6 has fixed the CI trigger, please rebase this pull request to enable CI jobs.

**Pull request merged**